src/State/Hydration/sagas.ts

```diff
@@ -1,6 +1,6 @@
 import * as types from './types';
 import { takeEvery, put, call } from 'redux-saga/effects';
-import { getSourceTiles } from './api';
+import { getSourceTiles, getDestinations } from './api';
 
 export function* handleSourceTiles() {
```

Comment:
we should probably meet to have a consistent naming structure such as prefixing this with "work" and api request naming

Reply:
added `work` as a prefix